### PR TITLE
Move out JaCoCo plugin from profiles tags

### PR DESCRIPTION
**Pull request by RamOwen99**
The JaCoCo plugin was inside from the profiles tag instead of the main
plugins tag, because all the profiles need to use JaCoCo

### CODE REVIEW

**Review by luisgalindoluna — APPROVED** [review]
Looks good

File: src/test/java/com/agilethought/atsceapi/service/UserServiceImplTest.java [hunk 1]
```diff
@@ -23,16 +27,21 @@
     @Mock
     private UserRepository userRepository;
 
+    @Mock
+    private MapperFacade userRepositoryMapper;
```
Comment (ivanflores15) [inline src/test/java/com/agilethought/atsceapi/service/UserServiceImplTest.java]: name of attribue it not matching with  type object. It does not make sense.

File: src/test/java/com/agilethought/atsceapi/service/UserServiceImplTest.java [hunk 2]
```diff
@@ -23,16 +27,21 @@
...
     private UserServiceImpl userService;
 
+    @Before
     public void setup() {
         MockitoAnnotations.initMocks(this);
     }
 
     @Test
-    public void getAllUsersTest(){
+    public void itShouldGetAllUsers(){
```
Comment (ivanflores15) [inline src/test/java/com/agilethought/atsceapi/service/UserServiceImplTest.java]: I prefer left previous name

**Review by ivanflores15 — CHANGES_REQUESTED** [review]

**Comment by ivanflores15** [comment]
your build is still having issue, and pipeline say no tests were run. It's something related to maven-surefire-plugin:2.22.2:test. we expected at least one test was run. 
image

File: src/test/java/com/agilethought/atsceapi/service/UserServiceImplTest.java [hunk 3]
```diff
@@ -1,12 +1,16 @@
 package com.agilethought.atsceapi.service;
 
-import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.*;
```
Comment (ivanflores15) [inline src/test/java/com/agilethought/atsceapi/service/UserServiceImplTest.java]: it is not recommended include all packages inside of library. just import packages that are used

**Review by ivanflores15 — CHANGES_REQUESTED** [review]

**Review by ivanflores15 — APPROVED** [review]
good job

**Review by EmiliaGuillen-dev — APPROVED** [review]
looks good

**Pull request merged**